scripts/newtab.js

```diff
@@ -0,0 +1,188 @@
+'use strict'
+
+var placeholders = [];
+var url = 'https://bwinton.github.io/whimsy/thumbnail-gifs.txt';
+var getting = browser.storage.sync.get('newtab');
+var listener = false;
+var tippyTopSites = require('tippy-top-sites');
+
+initialize();
+
+function onError(error) {
+  console.log(error);
+}
+function loadTopSites(){
+  //display top sites
+  browser.topSites.get().then((topSitesArray) => {
+    var grid = document.querySelector("#newtab-grid");
+    grid.innerHTML = "";
+    for (let topSite of topSitesArray) {
+      grid.innerHTML +=
+      "<div class='newtab-cell'><div class='newtab-site' draggable='true' type='history'>"+
+          "<a class='newtab-link' title="+topSite.title+" href="+topSite.url+">"+
+            "<span class='newtab-thumbnail thumbnail' style='background-size: cover; background-image: ;'></span>"+
```

Comment:
We should really also add [a contextmenu item](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/contextMenu) to let people copy the thumbnail url.  (It was one of our most requested features!)